bundles/io.openliberty.tools.eclipse.ui/src/io/openliberty/tools/eclipse/DevModeOperations.java

```diff
@@ -230,10 +230,21 @@ public void start(IProject iProject, String parms, String javaHomePath, ILaunch
             String cmd = "";
 
             if (buildType == Project.BuildType.MAVEN) {
-                cmd = CommandBuilder.getMavenCommandLine(projectPath, "io.openliberty.tools:liberty-maven-plugin:dev " + startParms,
+                cmd = CommandBuilder.getMavenCommandLine(projectPath, (runProjectClean == true ? " clean " : "" ) +  "io.openliberty.tools:liberty-maven-plugin:dev " + startParms,
                         pathEnv);
             } else if (buildType == Project.BuildType.GRADLE) {
-                cmd = CommandBuilder.getGradleCommandLine(projectPath, "libertyDev " + startParms, pathEnv);
+            	if (runProjectClean == true) {
+                    // Step 1: Run gradle --stop
```

Comment:
Can we have the gradle stop as part of the same command as the start?  So for example:
`./gradlew --stop && ./gradlew clean libertyDev` or even `./graldew --stop clean libertyDev`

Not sure if that last one will work, but the first should, I think.  By running the process separately, we have a system process showing in the debug UI that seems a bit awkward:
<img width="335" alt="Screenshot 2025-04-28 at 11 37 08 AM" src="https://github.com/user-attachments/assets/fb4684ce-5e28-468a-87ba-426e430adbf7" />